program/CORE-V IDE prelminary project proposal.md

```diff
@@ -0,0 +1,251 @@
+3. Embecosm will contribute CORE-V compiler tool chain and packaging or stabilization if necessary
+4. To be checked: Alibaba, CMC, Thales
+
+## Technical Project Leader(s) (TPLs)
+The following persons will investigate and resolve overall technical issues related to the IDE
+1. Alexander Fedorov  
+2. Hugh O'Keeffe or an alternative person from Ashling  
+
+## Project Manager (PM)
+The following is tentative:
+- Florian Zaruba of OpenHW staff will act as project manager
+- Jeremy Bennett of Embecosm and Duncan Bees of OpenHW staff will act as project management mentors.
+The project manager would be responsible, in conjuction with the TPLs, to create the project plan, collect requirements for features, track status and report status to the OpenHW TWG, and identify and resolve issues related to the successful outcome. 
+
+
+## Project Documents
+The following project documents will be created:
+- Preliminary Project Proposal (this document)
+- Project Proposal, an updated version of this document
+- Project Plan
+- Feature Description (initial version here: https://github.com/openhwgroup/core-v-sw/blob/master/planning/openhw-ide-requirement.md)
+- Build and Delivery Description
+
+
+
+
+## Virtual customer
+The project team would like to expore the concept of virtual customer to help provide suggestions on feature requirements, participate in ongoing backlog grooming, and review and feedback on CORE-V IDE as it is released. It is suggested that CMC (Hugh Politt-Smith) can act in this role as the IDE is used with the FPGA implementation of CV32E40P
+
+
+## Summary of requirements
+
+The present document gives a present snapshot of requirements and Preliminary Project Proposal, the requirements document https://github.com/openhwgroup/core-v-sw/blob/master/planning/openhw-ide-requirement.md will be used as the reference going forward. 
+
+### Introduction
+Use case list would be useful
+
+### Initial project requirements 
+
+1. Support for 64 bit Windows only
+2. Supplied as a ZIPped archive...no installer needed...just extract and ready to run
+3. �OpenHW IDE� splash-screen and "about" box with openHW Group graphics. 
+4. Set version to v0.0.1
+5. Build on latest, released Eclipse, CDT and GNU MCU  plug-ins (now known as Eclipse Embedded CDT (C/C++ Development Tools))
+6. Latest Embecosm CORE-V toolchain
+7. OpenOCD (Open On-Chip Debugger, with JTAG support) for CORE-V
+8. Simple Eclipse project ready-to-run on Genesys 2 board (e.g. LED toggle) including Opella-LD OpenOCD launch (Opella-LD is 	Ashling's low-cost, FTDI based probe not to be confused with the existing Opella-XD)
+9. OpenHW IDE getting started guide documentation with Genesys 2 board 
+
+
+### Future enhancements:
+
+10. Adding Linux support
+11. Automating the build to �pull� all the latest components from their respective repos etc
+12. Improved documentation and more board examples
+13. Various improvements
+
+
+
+## Explanation of why OpenHW should do this project
+

```

Comment:
Relate to the two charter goals

1. Building a thriving commercial ecosystem; and
2. Ensuring any open source tools are contributed and maintained upstream

Reply:
Comment to this effect added. We should note which upstream contributions are likely to be made.